guides/source/action_cable_overview.md

```diff
@@ -604,6 +604,13 @@ config.action_cable.log_tags = [
 ]
 ```
 
+**Content Security Policy**
+
+Rails 5.2 ships with a default Content Security Policy (CSP).
+In order to function properly the web socket connection needs to be whitelisted.
+Here is an example:
+`p.connect_src :self, :https, 'ws://localhost:5000'`
```

Comment:
The other time a port is mentioned it is 3000 which is the default port. Please change this to 3000.

This Code section is long please make it it’s own Code block with three back ticks and a space after the previous line.